liquidprompt

```diff
@@ -3376,6 +3489,13 @@ _lp_default_theme_prompt_data() {
         LP_VCS=
     fi
 
+    # Should be called after the dev env group above.
+    if _lp_dev_env_color ; then
+        LP_DEV_ENV="$lp_dev_env_color"
```

Comment:
Shouldn't this lead with a space?
```suggestion
        LP_DEV_ENV=" $lp_dev_env_color"
```
I see that the space is added on line 3287, but I think it should be added here instead like the other functions do, so other themes using the function can get it without any opinionated spacing.